src/components/Dialogs/ImportSettingsDialog.vue

```diff
@@ -37,7 +37,7 @@ async function submit() {
   if (!isFormValid.value) return
 
   window.localStorage.setItem('vuetorrent_webuiSettings', settings.value)
-  window.location.reload()
+  location.reload()
```

Comment:
You sure this is available in this context?

Reply:
We already use that when importing / resetting settings. It's just to align to a single syntax

Reply:
Both window and location are in the global context of the browser